lnrpc/lightning.yaml

```diff
@@ -121,6 +121,9 @@ http:
     - selector: lnrpc.Lightning.UpdateChannelPolicy
       post: "/v1/chanpolicy"
       body: "*"
+    - selector: lnrpc.Lightning.UpdateNodeAnnouncement
+      post: "/v1/nodeannouncement"
```

Comment:
I guess that for now everything is simply under `/v1/`? better naming is welcomed